_includes/head.html

```diff
@@ -26,23 +26,29 @@
 
   <!-- title -->
   {% if page.title %}
-    <title>{{ page.title }}</title>
-    <meta property="og:title" content="{{ page.title | xml_escape }}" />
-    <meta name="twitter:title" content="{{ page.title | xml_escape }}">
+  <title>{{ page.title }}</title>
+  <meta property="og:title" content="{{ page.title | xml_escape }}">
+  <meta name="twitter:title" content="{{ page.title | xml_escape }}">
   {% else %}
-    <title>{{ site.title }}</title>
-    <meta property="og:title" content="{{ site.title | xml_escape }}" />
-    <meta name="twitter:title" content="{{ site.title | xml_escape }}">
+  <title>{{ site.title }}</title>
+  <meta property="og:title" content="{{ site.title | xml_escape }}">
+  <meta name="twitter:title" content="{{ site.title | xml_escape }}">
+  {% endif %}
+
+  {% if site.branch and site.branch != 'master' %}
+  {% capture canonical_base_url %}https://federalist.18f.gov/preview/18F/college-choice/{{ site.branch }}{% endcapture %}
+  {% else %}
+  {% assign canonical_base_url = site.url %}
   {% endif %}
 
   <!-- url -->
-  <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.url }}">
-  <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: site.url }}">
+  <!-- <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.url }}"> -->
+  <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: canonical_base_url }}">
```

Comment:
Is there a benefit to keeping og:url? Should probably just remove it, since I think it might be causing similar issues to the canonical when individual school pages are shared on FB